build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -488,22 +489,28 @@ export class FluidPackageCheck {
     private static checkLintScripts(pkg: Package, fix: boolean) {
         let fixed = false;
         if (pkg.getScript("build")) {
-            const hasPrettier = pkg.getScript("prettier");
-            const lintChildren = hasPrettier ? ["prettier", "eslint"] : ["eslint"];
-            if (this.checkChildrenScripts(pkg, "lint", lintChildren, false, fix)) {
-                fixed = true;
+            // const hasPrettier = pkg.getScript("prettier");
+            const hasLint = pkg.getScript("lint") === "npm run prettier && npm run eslint";
```

Comment:
I think we want to accept the lint script if it is either `npm run prettier && npm run eslint` or `npm run eslint` to make our current situation work better.

Longer term, I think we probably just want to make sure the lint script at least includes a call to `eslint` and `prettier`, but not care about the order, etc., but for now we need to ensure the script ends with the eslint call so the command line arguments we pass down for it are applied correctly.

That said, we could potentially do something like just verify that the script *ends* with `npm run eslint`, so it is accommodating of other linter scripts a package may want to run earlier. E.g. one of the sample apps I've been working in lately runs something like `npm run prettier && npm run good-fences && npm run eslint`.

Reply:
Another thing we want to consider longer term is allowing packages to run their linter scripts differently. E.g. running all of the checks in parallel like `concurrently npm:prettier npm:eslint`. But I don't think we have a good path for supporting this right now. Just something to consider as we make improvements in this space 🙂

Reply:
I made changes based on the long-term plan that you mentioned, so that it would not care about the order of the script but primarily check if `eslint` is included in the script. 

If not, we will be adding `eslint` and `eslint:fix` (for `lint` and `lint:fix` respectively) as the last element of the array `lintChildren` & `lintFixChildren` which will be concatenated as `expected` scripts for our `package.json`